### DISCUSSION TITLE
Consider publish compiled binary for releases

### DISCUSSION THREAD

**Discussion by Ljzn** [near/nearcore#5905] [Ideas] [answered]
The build time of nearcore is too long, it would be easier to use if we could have a binary release. For most common archs like linux.

**Comment by frol** [comment, marked as answer]
Yeah, that makes total sense, and we already have discussed it with @mhalambek some time ago, so I created an issue for that #5983

**Comment by maxzaver** [comment]
If we publish binary releases then we would also need to have release signing mechanism. We should consult John Smith.

**Comment by maxzaver** [comment]
Moved conversation into a private stream for security reasons: https://near.zulipchat.com/#narrow/stream/308695-nearinc.2Fprivate/topic/Binary.20release.20signing/near/267793192
